### PR TITLE
[BD-6222] Add strong badge theme to bpk-mixins

### PR DESCRIPTION
**Pull request by giriawu**
Figma: https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=6099%3A2722

In the new design of the hotel card, we have made extensive use of `BpkBadge` which is white color on a black background.
But this theme style is not yet well supported on the web (they are already supported on iOs and Android).
So we added it to `bpk-component-badge` as the base theme - **strong**, for better consistency support and user experience.

Related PR in backpack: https://github.com/Skyscanner/backpack/pull/2532

## Snapshot
image

image
Remember to include the following changes:

- [x] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack-foundations/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+**Changed**
```
Comment (jasonhuang-sky) [inline UNRELEASED.md:1]: It seems that would better be `Added`?

File: UNRELEASED.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+- `bpk-mixins`
+    - The `strong` badge theme has been added, which aligns to our mobile implementations and meets the needs of users.
```
Comment (jasonhuang-sky) [inline UNRELEASED.md:4]: what does `aligns to our mobile implementations` mean?

Comment (giriawu) [reply UNRELEASED.md:4]: What I mean is that this theme color is already added on mobile for IOS and Android, so it is necessary to add it on the web as well.

Comment (giriawu) [reply UNRELEASED.md]: Have modified the sentence to make it clearer 😇

**Review by jasonhuang-sky — APPROVED** [review]

**Review by anambl — APPROVED** [review]
LGTM!

**Pull request merged**